crates/iota-light-client/README.md

```diff
@@ -20,45 +20,62 @@ The light client requires a config file and a directory to cache checkpoints, an
 
 ## Setup
 
-The config file for the light client takes a URL for a full node, a directory (that must exist) and within the directory to name of the genesis blob for the IOTA network.
+The config file for the light client takes a URL for a full node, a directory to store checkpoint summaries (that must exist) and within the directory the name of the genesis blob for the IOTA network.
 
 ```
-full_node_url: "https://api.testnet.iota.cafe:443"
-checkpoint_summary_dir: "checkpoints_dir"
-genesis_filename: "genesis.blob"
+rpc_url: "https://api.testnet.iota.cafe"
+graphql_url: "https://graphql.testnet.iota.cafe"
+checkpoints_dir: "checkpoints_testnet"
+# If not provided, make sure genesis.blob exists in checkpoints_dir
+genesis_blob_download_url: "https://dbfiles.testnet.iota.cafe/genesis.blob"
+sync_before_check: false
+object_store_url: "https://checkpoints.testnet.iota.cafe"
+archive_store_config:
+  object-store: "S3"
+  aws-endpoint: "https://archive.testnet.iota.cafe"
+  aws-virtual-hosted-style-request: true
+  no-sign-request: true
+  aws-region: "weur"
+  object-store-connection-limit: 20
+
 ```
 
-The genesis blob for the IOTA mainnet can be found here: https://github.com/iotaledger/iota-genesis/blob/main/mainnet/genesis.blob
+The genesis blob for the IOTA testnet can be found here: https://dbfiles.testnet.iota.cafe/genesis.blob. Download and place it inside the checkpoint directory.
 
 ## Sync
 
 Every day there is a need to download new checkpoints through sync by doing:
 
 ```
-$ iota-light-client --config light_client.yaml sync
+$ iota-light-client --config testnet.yaml sync
 ```
 
-Where `light_client.yaml` is the config file above.
+Where `testnet.yaml` is the config file above.
 
 This command will download all end-of-epoch checkpoints, and check them for validity. They will be cached within the checkpoint summary directory for use by future invocations.
 
+Internally, sync works in two steps. It first downloads the end-of-epoch checkpoint numbers into the `checkpoints.yaml` file (which needs to be present in the checkpoint summaries directory). Next, it downloads the corresponding checkpoint summaries.
```

Comment:
`(which needs to be present in the checkpoint summaries directory)` is not needed anymore?

Reply:
Just sounded a bit as if one has to place it there before, but that's not explained here, so I guess it's fine